packages/apps/dev-wallet/src/pages/unlock-profile/styles.css.ts

```diff
@@ -0,0 +1,17 @@
+import { style } from '@vanilla-extract/css';
+import { tokens } from '@kadena/react-ui/styles';
+
+export const passwordContainer = style([
+  {
+    marginTop: tokens.kda.foundation.spacing.md,
```

Comment:
For such margins and paddings, maybe better to use atoms?

Reply:
updated